test/unit/test_teams_views/data/exp_team_dist.txt

```diff
@@ -0,0 +1,17 @@
+[u]Same 7 series and same active team/bench distribution[/u]
+Number of unique teams: 3
+
+Same team chosen by:
+[list][/list]
+[u]Same 7 series but different active team/bench distribution[/u]
```

Comment:
is this an effective set of test data if we're not showing any results here?

Reply:
Now that I think about it, not really. I'll add another team with the same active and bench as another team to the test.